pandas/tests/groupby/test_groupby_shift_emptydf.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pytest
+
+from pandas import (
```

Comment:
pls don't make a new file, look for a similar test in this directory and put it near.